x/dex/types/errors.go

```diff
@@ -10,39 +10,194 @@ import (
 
 //nolint:all
 var (
```

Comment:
this formatting is a bit odd

Reply:
I like it better. Golang seems like the only language in the world where the community doesn't mind run-on lines.